content/rc/how-to/flush-db.md

```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
+To flush data from a CRDB, use the flush command in the crdb-cli.
+
+1. To find the ID of the CRDB, run:
+
+    ```src
+    crdb-cli crdb list
+    ```
```

Comment:
Maybe it worth printing example of the output and where the ID is located

Reply:
OK